azure.dockerfile

```diff
@@ -0,0 +1,31 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
+
+ARG CLI_VERSION
+
+# Metadata as defined at http://label-schema.org
+ARG BUILD_DATE
+
+LABEL maintainer="Microsoft" \
+      org.label-schema.schema-version="1.0" \
+      org.label-schema.vendor="Microsoft" \
+      org.label-schema.name="Azure CLI" \
+      org.label-schema.version=$CLI_VERSION \
+      org.label-schema.license="MIT" \
+      org.label-schema.description="The Azure CLI is used for all Resource Manager deployments in Azure." \
```

Comment:
This description added 6 years ago by https://github.com/Azure/azure-cli/pull/4696 is no longer accurate, given we now support Microsoft Graph and other data-plane operations as well.

Reply:
I found a new description in the repo `A great cloud needs great tools; we're excited to introduce Azure CLI, our next generation multi-platform command line experience for Azure.`. Do we have any alternative for this?